src/renderer/components/top-nav/top-nav.js

```diff
@@ -333,8 +339,8 @@ export default defineComponent({
     navigate: function (route) {
       this.$router.push('/' + route)
     },
-    updateSearchInputText: function (text) {
-      this.$refs.searchInput.updateInputData(text)
+    updateSearchInputText: function ({ detail: { query } }) {
```

Comment:
Let me start with sazing that I have no position in the FreeTube project, but there is one note from me. If the interface of the ` updateSearchInputText` function should be changed - it's now object instead of string - the function should probably be renamed to better signify the use case.